File: papi/lista2/exercicio6.py
import math

print("Aqui calcularemos a área de uma circunferência.")

while True:
    try:
        raio = float(input("Por favor favor informe o raio da circunferência em centímetros - use o ponto como separador decimal: "))
        
        if raio < 0:
            raise ValueError

        break
    except ValueError:
        print("Por favor informe um número válido e maior que zero.")

print("A área da circunferência em centímetros é", raio * raio * math.pi)

File: papi/lista2/exercicio6.kt
package lista2

fun main() {
    var raio: Double
    while (true) {
        try {
            print("Digite o raio em centímetros, usando o ponto como separador decimal: ")
            val raioString = readLine() 
            raio = raioString!!.toDouble()
            break
        } catch (e: Exception) {
            println("Por favor, insira um valor válido para o raio, usando o ponto como separador decimal")
        }
    }
    val area = Math.PI * Math.pow(raio, 2.0)
    println("A área do círculo com um raio de $raio centímetros é $area")
}   